### DISCUSSION TITLE
Gteh.org pool for ETHASH and ETCHASH

### DISCUSSION THREAD

**Discussion by slappy6212** [RainbowMiner/RainbowMiner#2533] [General]
Hi, Is it possible to add the following pool Gteh.org please ?

**Comment by RainbowMiner** [comment]
Interesting pool, I will try to add it.

**Reply by slappy6212** [reply]
yes we can change the mining coin within ETHASH algo from the web. It's practicle when at work. XPB coin give us great results since couples of weeks.

**Comment by slappy6212** [comment]
Hi,
Thanks for adding Gteh.
I entered my id, my key, and configure the rest in pool config but i dont see Gteh in all pools tab.
I missed something ?

**Comment by RainbowMiner** [comment]
Yes, you need to specify which coins to mine in parameter "CoinSymbol" in pools.config.txt

```
    "Account_Id": "yourAccountId",
    "API_Key": "yourAPIkey",
...
    "CoinSymbol": "CAU,ETC,XPB,EGAZ",
```

**Reply by slappy6212** [reply]
Id, api and Coins already there, Gteh not showing in all pool...look like the pool is disable somewhere.

**Comment by slappy6212** [comment]
Still not showing. Are you having Gteh on yours ?

**Reply by RainbowMiner** [reply]
In fact, I have. Could you please upload a Debug file? I'll have a look then.

**Comment by slappy6212** [comment]
[RainbowMiner_2023-12-08.txt](https://github.com/RainbowMiner/RainbowMiner/files/13614062/RainbowMiner_2023-12-08.txt)

**Reply by slappy6212** [reply]
Thanks

**Reply by RainbowMiner** [reply]
Oh, I ment "Debug file" - just click "Debug file" on http://localhost:4000
This will contain all vital information.

**Comment by slappy6212** [comment]
[debug_2023-12-08.zip](https://github.com/RainbowMiner/RainbowMiner/files/13615213/debug_2023-12-08.zip)

**Comment by slappy6212** [comment]
[debug_2023-12-08_2.zip](https://github.com/RainbowMiner/RainbowMiner/files/13622217/debug_2023-12-08_2.zip)

**Reply by slappy6212** [reply]
Sorry the first one was corrupt

**Reply by RainbowMiner** [reply]
Ok, gotcha: please empty "Algorithm" in pools.config.txt in "Gteh" section.

It currently contains the pool name:
```
      "Algorithm": [
        "Gteh"
      ],
```

**Comment by slappy6212** [comment]
No. I already tried to remove it because it was a strange value...no luck still missing GTEH.

**Comment by slappy6212** [comment]
This morning i did a fresh install, and tried using only GTEH pool...no luck. Also as a general explanation why sometimes miners are disable for certain algo but available for others. Exemple  BZMINER is not available for NEXAPOW as it should In accordance with their web site. Is there a way to force it ? Also Dynex is now back for AMD users with latest buid of BZMINER. What files should i update on top of the BIN Folder to allow Dynex + BZminer if i dont want to wait for your next update ?

**Comment by slappy6212** [comment]
Maybe its related to my api key. I can see it in the config but still not showing in all pool.
Can you share your settings for gtech please ?

**Comment by slappy3** [comment]
Update: Still not working but at leat one of my rig having the pool turning ON but with only 1 coin (BTN) is available. I progress LOL
Here is my actual settings: except the id and the key.
"Gteh": {
    "Account_Id": "xxxxxxxxxxxxxxxxxxxxx",
    "API_Key": "xxxxxxxxxxxxxxxxxxxx",
    "UseWorkerName": "Xigmatek",
    "ExcludeWorkerName": "",
    "EnableMiningSwitch": "1",
    "Worker": "$WorkerName",
    "Penalty": "0",
    "DataWindow": "",
    "StatAverage": "",
    "StatAverageStable": "",
    "MaxMarginOfError": "100",
    "SwitchingHysteresis": "",
    "MaxAllowedLuck": "",
    "MaxTimeSinceLastBlock": "",
    "MaxTimeToFind": "",
    "Region": "",
    "SSL": "0",
    "BalancesKeepAlive": "",
    "Algorithm": "",
    "ExcludeAlgorithm": "",
    "CoinName": "",
    "ExcludeCoin": "",
    "CoinSymbol": "CAU,ELH,XPB,AVS,OCTA,ETHW,NYBC,REDEV2,BTN,HYP,DOGETHER,EGAZ,ZIL",
    "ExcludeCoinSymbol": "",
    "CoinSymbolPBM": "",
    "MinerName": "",
    "ExcludeMinerName": "",
    "EnableAutoCoin": "0",
    "EnablePostBlockMining": "0",
    "FocusWallet": "",
    "AllowZero": "0"
  },
